src/main/java/com/example/coursework2/JavaQuestionController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.coursework2;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+    private final JavaQuestionService javaQuestionService;
+
+    public JavaQuestionController(JavaQuestionService javaQuestionService) {
```

Comment:
```suggestion
    public JavaQuestionController(QuestionService javaQuestionService) {
```
инъекцию надо делать по интерфейсу